BUILD.md

```diff
@@ -142,141 +121,91 @@ At this point, you can build the solution from the command line or open the
         libXrandr-devel wayland-devel
 ```
 
-
 ### Linux Build
-
 The general approach is to run CMake to generate make files. Then either run
 CMake with the `--build` option or `make` to build from the command line.
 
-
 #### Use CMake to Create the Make Files
-
-Change your current directory to the top of the cloned repository directory,
-create a build directory and generate the make files.
-
+Change your current directory to the top level directory for the cloned
+GFXReconstruct repository, create a build folder, and generate the make
+files.
 ```
 cd gfxreconstruct
 mkdir build
 cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Debug
 ```
 
-Use `-DCMAKE_BUILD_TYPE` to specify a Debug or Release build.
+Use `-DCMAKE_BUILD_TYPE` to specify a `Debug` or `Release` build profile.
 
 #### Build the Project
+Run either the `make` or `cmake --build .` command from the build directory
+to build the GFXReconstruct project source.
 
-You can just run `make` to begin the build.
-
-To speed up the build on a multi-core machine, use the `-j` option for `make`
-to specify the number of cores to use for the build. For example:
-
+For faster builds on multi-core systems with `make`, use `make -j` to
+specify the number of cores to use for the build. For example:
 ```
 make -j4
 ```
 
-You can also use
-
-```
-cmake --build .
-```
-
-If your build system supports ccache, you can enable that via CMake option `-DUSE_CCACHE=On`
-
+For build systems that support ccache, enable it with the CMake
+`-DUSE_CCACHE=On` option.
 
 ## Android
-
-### Android Preparation
-
-Install the required tools for Linux and Windows covered above, then add the
-following:
-
+### Android Develpment Requirements
  * The latest version of [Android Studio](https://developer.android.com/studio/) with additional items:
    * The [Android Platform tools](https://developer.android.com/studio/releases/platform-tools) for your specific platform
    * The [Android NDK](https://developer.android.com/ndk/guides/)
-   * [Android SDK 23 (6.0 Marshmallow) or newer](https://guides.codepath.com/android/installing-android-sdk-tools)
+   * [Android SDK 24 (7.0 Nougat) or newer](https://guides.codepath.com/android/installing-android-sdk-tools)
  * Set the following `ANDROID_*` environment variables appropriately in your user environment if building outside of Android Studio:
    * `ANDROID_NDK_HOME`
    * `ANDROID_SDK_HOME`
 
-### Android Build Process
-
-In the following snippets you will see the following defines.  Replace them with the item described next to them in the table:
-
-
-| Tag | Replace With  |
-|---|---|
-| `{gfxreconstruct_root}` | Substitute in the path to the GFXReconstruct source folder on your system |
-
-
-#### Building With Gradle
-
-To perform a debug build from the command line with Gradle, run the following command from the
-`{gfxreconstruct_root}/android` folder:
+### Android Build
+#### Building with Gradle
+Run the following command from the `gfxreconstruct\android` folder to perform a
```

Comment:
I thought Android uses LInux-style path separators, e.g. `gfxreconstruct/android` instead of `gfxreconstruct\android`